dir.props

```diff
@@ -91,7 +91,7 @@
       <ExpectedPrerelease>$(CoreFxExpectedPrerelease)</ExpectedPrerelease>
     </ValidationPattern>
     <ValidationPattern Include="ExternalVersions">
-      <IdentityRegex>^(?i)System\.Data\.SqlClient$</IdentityRegex>
+      <IdentityRegex>^(?i)((System\.Data\.SqlClient)|(System\.IO\.Compression)|(System\.Net\.Http)|(System\.Net\.Security))$</IdentityRegex>
```

Comment:
As I sent en an offline email I would like to limit this to only partial Compression but for the other 2 I don't think we should do exclude these from the open builds.